cmd/clusterctl/client/config/providers_client.go

```diff
@@ -98,13 +98,19 @@ const (
 // IPAM providers.
 const (
 	InClusterIPAMProviderName = "in-cluster"
+	NutanixIPAMProviderName   = "nutanix"
```

Comment:
```suggestion
	NutanixIPAMProviderName   = "nutanix-cloud-native-nutanix"
```

To follow the conventions at https://main.cluster-api.sigs.k8s.io/clusterctl/provider-contract.html#adding-a-provider-to-clusterctl

Reply:
We already have an infra provider simply called nutanix at https://github.com/kubernetes-sigs/cluster-api/blob/main/cmd/clusterctl/client/config/providers_client.go#L55 and it would be great if we could use the same name for these other provider types for consistency 🙏

Reply:
Good point. Maybe it's fine if a GitHub org already "claimed" the name via another provider

Reply:
Would make sense. Fine for me if we have an agreement :-)

Reply:
@fabriziopandini @vincepri WDYT?

Reply:
I think we can allow this because https://github.com/nutanix is owned by the same org that owns https://github.com/nutanix-cloud-native.

Do you mind updating https://cluster-api.sigs.k8s.io/clusterctl/provider-contract.html#providers-github-org-prefix with something like

In case the entity proposing the new providers owns different github organizations, it is also possible to Provider's GitHub org prefix (in this case it is up to this entity to avoid or address conflicts).

Reply:
I'm not sure we need to update any docs. Can't we just agree as we (Nutanix) already claimed the `nutanix` provider name for infra provider as mentioned above?

Reply:
I've added some docs as requested. Please review when you get a chance 🙏